rpc/types_contract_test.go

```diff
@@ -17,7 +17,7 @@ func TestContractClass_UnmarshalValidJSON_Successful(t *testing.T) {
 		t.Fatal("should be able to read file", err)
 	}
 
-	contractClass := ContractClass{}
+	contractClass := DepcreatedContractClass{}
```

Comment:
Could we change the name of this test to `TestDeprecatedContractClass_UnmarshalValidJSON_Successful` instead?
Should we add another test for the new `ContractClass` type?